projects/storefrontapp-e2e-cypress/cypress/helpers/cart.ts

```diff
@@ -321,3 +288,26 @@ export function outOfStock() {
   cy.get('cx-add-to-cart .quantity').should('contain', 'Out of stock');
   cy.get('cx-add-to-cart cx-add-to-cart button').should('not.exist');
 }
+
+export const cartUser = {
+  user: 'standard',
+  registrationData: {
+    firstName: 'Winston',
+    lastName: 'Rumfoord',
+    password: 'Password123.',
+    titleCode: 'mr',
+    email: generateMail(randomString(), true),
+  },
+};
+
+export function registerCartUser() {
+  cy.visit('/login/register');
+  register({ ...cartUser.registrationData });
+  cy.url().should('not.contain', 'register');
+}
+
+export function loginCartUser() {
```

Comment:
### For authors

Accept good ideas from reviewers. Don't take them personally and don't get attached to your solution, when better alternatives are available.

Reply:
![Butterfly_man_meme_about_feeling_attacked_by_code_reviewers-300x300](https://user-images.githubusercontent.com/6259079/144898734-1aa7c41a-bea9-4bc4-9cba-a779d9eddca0.jpeg)